docs/examples/tiledata/group.js

```diff
@@ -0,0 +1,65 @@
+// @flow strict
+import { useState, type Node } from 'react';
+import { TileData, Flex } from 'gestalt';
+
+export default function Example(): Node {
+  const DataSources = [
+    {
+      id: 'data-1',
+      name: 'MAU',
+      value: '100M',
+      color: '01',
+      tooltip: 'Monthly active users',
+    },
+    {
+      id: 'data-2',
+      name: 'WAU',
+      value: '80M',
+      color: '02',
+      tooltip: 'Weekly active users',
+    },
+    {
+      id: 'data-4',
+      name: 'DAU',
+      value: '10M',
+      color: '04',
+      tooltip: 'Daily active users',
+    },
+  ];
+
+  // add all elements to our map
+  const selectedMap = DataSources.reduce((acc, tile) => {
```

Comment:
Is a map the best data structure for this? Would an array of selected ids be simpler?